man/sd_pid_get_owner_uid.xml

```diff
@@ -169,54 +169,57 @@
   <refsect1>
     <title>Description</title>
 
+    <para><function>sd_pid_get_owner_uid()</function> may be used to
+    determine the Unix UID (user identifier) which owns the login
+    session or systemd user unit of a process identified by the
+    specified PID. For processes which are not part of a login session
+    and not managed by a user manager, this function will fail with
+    <constant>-ENODATA</constant>.</para>
+
     <para><function>sd_pid_get_session()</function> may be used to
     determine the login session identifier of a process identified by
     the specified process identifier. The session identifier is a
-    short string, suitable for usage in file system paths. Note that
-    not all processes are part of a login session (e.g. system service
-    processes, user processes that are shared between multiple
-    sessions of the same user, or kernel threads). For processes not
-    being part of a login session, this function will fail with
-    <constant>-ENODATA</constant>. The returned string needs to be freed with the libc
-    <citerefentry
+    short string, suitable for usage in file system paths. Please
+    note this function is not generally useful, as many user processes
```

Comment:
I agree that the initial API design is not useful for some of the original purposes, in particular for session and lingering management, with the move to per-user dbus and moving of processes of of the session scope. Nevertheless, I don't think it's correct to say that that part of the API is "not generally useful". You could say that if the function stopped working. It does, but just what it does cannot be used for some stuff.

Instead, I'd prefer to see some text like "Note that this function is applicable only to processes
that actually are part of a user session. Many user processes are started from their systemd user
manager... [the rest of the text is OK]".

Also, I see your point about de-emphasizing sd_pid_get_session. But you need to keep the man pages
internally consistent. The script to regenerate `man/rules/meson.build` has various checks that must pass.